### DISCUSSION TITLE
Не могу подключить на nuxt3.

### DISCUSSION THREAD

**Discussion by sigyn13** [yandex-maps-unofficial/vue-yandex-maps#394] [Q&A (2.x)]
Привет! Подключила на другом проекте эту либу, все отлично работает. Щас на другом проекте хочу перейти на новую версию 2.1.4 с старой, сделала все по доке и аналогично другому проекту, но почему то получаю ошибку: 
[warn] [nuxt] Some plugins are not exposing a function and skipped
Подскажите, что я делаю не так? Помогите, пожалуйста. 
Вот так подключаю в nuxt.config.ts:
` modules: ['nuxt-svgo', '@pinia/nuxt', 'vue-yandex-maps/nuxt'],
  yandexMaps: {
    apikey:
      process.env.NUXT_PUBLIC_YANDEX_MAP_API_KEY ??
      'xxxx',
  },`
  
  package.json: 
  `  "devDependencies": {
    "@babel/eslint-parser": "^7.19.1",
    "@babel/eslint-plugin": "^7.19.1",
    "@nuxtjs/eslint-config-typescript": "^11.0.0",
    "@nuxtjs/eslint-module": "^3.1.0",
    "@typescript-eslint/eslint-plugin": "^5.38.1",
    "@typescript-eslint/parser": "^5.38.1",
    "cypress": "^12.5.0",
    "eslint": "^8.24.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-nuxt": "^4.0.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-simple-import-sort": "^8.0.0",
    "eslint-plugin-vue": "^9.5.1",
    "nuxt": "^3.2.2",
    "nuxt-svgo": "^1.0.1",
    "postcss-html": "^1.5.0",
    "prettier": "^2.7.1",
    "sass": "^1.55.0",
    "sass-loader": "^13.0.2",
    "stylelint": "^14.13.0",
    "stylelint-config-prettier": "^9.0.3",
    "stylelint-config-recommended-vue": "^1.4.0",
    "stylelint-config-standard": "^28.0.0",
    "typescript": "^4.8.4",
    "vite-plugin-eslint": "^1.8.1"
  },
  "dependencies": {
    "@chenfengyuan/vue-qrcode": "2",
    "@pinia/nuxt": "^0.4.6",
    "@shimyshack/uid": "^0.1.9",
    "@vue/composition-api": "^1.7.1",
    "@vuepic/vue-datepicker": "^8.6.0",
    "eslint-plugin-autofix": "^1.1.0",
    "iframe-resizer": "^4.3.2",
    "path": "^0.12.7",
    "pinia": "^2.0.28",
    "qrcode": "1",
    "vue-bind-once": "^0.2.1",
    "vue-composable": "^1.0.0-beta.24",
    "vue-yandex-maps": "^2.1.4"
  }`

**Comment by daniluk4000** [comment]
Привет. 

Подключение правильное... На моих проектах версия работает нормально.

Могу предположить, что у вас древнющая версия Nuxt. Не вижу нигде версии vue, но у нас минималка 3.3. Предлагаю вам пообновлять зависимости (и пересобрать локфайл), так как наш проект рекомендует для работы последние версии vue/nuxt.

**Reply by sigyn13** [reply]
Спасибо, попробую!

**Comment by sigyn13** [comment]
Я так и не смогла найти решение. При пересборке yarn.lock у меня проект падает, видимо yarn шалит. Может быть позже вернусь к этому, пока придется остаться на старой версии 1.0.7.

**Reply by daniluk4000** [reply]
¯\_(ツ)_/¯

Так или иначе Nuxt древний, я советую заняться обновлением, там и уязвимостей уже может быть полно. А то, что заработают яндекс карты, это вторичное)

**Reply by sigyn13** [reply]
Так в том и дело, что не древний, последняя версия стоит. Проект не я писала просто, он большой и довольно сложный. Скорее всего что-то из зависимостей друг с другом не оч дружит, а разобраться ничего не сломав оч тяжело(

**Reply by daniluk4000** [reply]
В package.json у вас nuxt версии ^3.2.2, если пересборка локфайла что-то ломает, то это точно не новая версия

Меня еще смущают библиотеки по типу @vue/composition-api, path, vue-composable, sass-loader (не нужен в vite), по ощущениям зависимости давно не обновлялись и не пересматривались)

**Reply by daniluk4000** [reply]
Я просто еще не могу понять версию vue, минимальная у нас 3.3, Nuxt я не тестировал, но версии из вашего package.json больше года

**Reply by sigyn13** [reply]
Да, там такая каша. У меня тоже большие сомнения насчет @vue/composition-api, там и vue почему-то двух версий внутри. В общем беда с зависимостями, как будет время буду разбираться что к чему.
